src/ws/cockpit.service.in

```diff
@@ -4,8 +4,8 @@ Documentation=man:cockpit-ws(8)
 Requires=cockpit.socket
 
 [Service]
-ExecStartPre=@sbindir@/remotectl certificate --ensure --user=root --group=@group@ --selinux-type=@selinux_config_type@
+ExecStartPre=@sbindir@/remotectl certificate --ensure --user=@cockpit_user@ --group=@cockpit_group@ --selinux-type=@selinux_config_type@
```

Comment:
Good point. This is caused by https://www.gnu.org/software/autoconf/manual/autoconf-2.68/html_node/Installation-Directory-Variables.html#Installation-Directory-Variables

Changed to avoid using cofigure.ac